### ISSUE TITLE
Evaluate Allen's user interface drafts

### ISSUE THREAD

**Issue by frainz-de** [getting-things-gnome/gtg#436]
As this project is specifically about GTD, it might be useful to have a look at David Allen's dream user interface which he revealed in a [blog post](https://gettingthingsdone.com/2019/07/david-allen-on-the-best-software-for-gtd/) in 2019. There is also a [short speech](https://www.youtube.com/watch?v=2c_RuXVujho) by him where he describes the interface.
Many of the slides are probably out of the scope of this project, but some might be good inspiration for future improvements.
One problem might be the license of the provided slides. The post says 

> This was originally legally protected IP; but I decided to just make it “open source” now in case any of you out there could make it happen.

but there doesn't seem to be an explicit license.

Sorry if an issue is the wrong place for this, maybe we should put it into the wiki?

**Comment by nekohayo** [comment]
Yeah I saw those slides but didn't quite know what actionable improvements to derive from them. Have you looked at our existing list of issues tagged "enhancement"? And if there are specific things you like from David's brainstorming slides that could be turned into actionable feature requests for GTG, then you can file them as tickets (and those will dependent on the availability of people interested in showing up to implement them). Otherwise, "We should look at David's 19 slides of brainstorming" is too vague of a request, and if nothing comes out of this in the next few months, this might be closed to keep the issue tracker clean and actionable.

Note also that GTG is GTD-capable, but not limited to nor "enforcing" GTD, it is meant to be as flexible as possible while also as simple as possible for people to learn. And my cursory glance at the slides made me feel like David tends to overcomplicate/overengineer things...

**Comment by frainz-de** [comment]
My reason to post them was that I thought they could provide inspiration for more concrete enhancements but they might just be too unclear for that. I think the best would be to link to them somewhere in CONTRIBUTING.md but you can also just close the issue if you think that's a bad idea.

**Comment by digitalethics** [comment]
You could change the label from _question_ to _discussion_ and keep this issue open for longer. I will take some time to extract actionable ideas from the PDFs and the speech. There are two very important **key messages** from the short video that I can already mention now: 
- [dashboard](https://github.com/getting-things-gnome/gtg/issues/429)
- bidirectional linking
